c-scape/src/fs/access.rs

```diff
@@ -32,3 +32,15 @@ unsafe extern "C" fn faccessat(
         None => -1,
     }
 }
+
+#[no_mangle]
+unsafe extern "C" fn euidaccess(pathname: *const c_char, amode: c_int) -> c_int {
+    libc!(libc::euidaccess(pathname, amode));
+    faccessat(libc::AT_FDCWD, pathname, amode, libc::AT_EACCESS)
+}
+
+#[no_mangle]
+unsafe extern "C" fn eaccess(pathname: *const c_char, amode: c_int) -> c_int {
+    libc!(libc::eaccess(pathname, amode));
+    faccessat(libc::AT_FDCWD, pathname, amode, libc::AT_EACCESS)
```

Comment:
Yeah, they're aliases. I'll just make it a call to `euidaccess` until Rust gets proper symbol aliases. I did it this way to avoid extra overhead, but the compiler will probably take care of it anyways.